docs/integrations/nbext.md

```diff
@@ -1,47 +1,53 @@
 
-<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button" >
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button">
 
 #### Valid API key
 
 If the key is valid you will be notified with the following alert.
 
-<img src="https://i.imgur.com/UHvSihx.png" class="screenshot" alt="valid api key prompt" >
+<img src="https://i.imgur.com/UHvSihx.png" class="screenshot" alt="valid api key prompt">
 
 #### Error with API key
 
 If the entered API key is invalid you will get following error.
 
-<img src="https://i.imgur.com/9WaVkTR.png" class="screenshot" alt="api key error prompt" >
+<img src="https://i.imgur.com/9WaVkTR.png" class="screenshot" alt="api key error prompt">
 
 #### Successful Commit
 
 Once the API key has been validated, you can start committing to [Jovian](https://jovian.ml?utm_source=docs) by clicking `Commit` button. Once the Notebook has been committed successfully you will get the confirmation message with the link where the Jupyter Notebook has been uploaded to, you can use the copy button to get the link to the share the notebook.
 
-<img src="https://i.imgur.com/4GoqzER.png" class="screenshot" alt="jovian commit success prompt" >
+<img src="https://i.imgur.com/4GoqzER.png" class="screenshot" alt="jovian commit success prompt">
 
-## Commit Dropdown to commit with more options
+## Commit with more options
 
-This makes use of [jovian.commit's](../jvn/commit) parameters to enable the user to commit with preferences like secret notebook, new notebook project, to add output and artifact files .....
+This makes use of [jovian.commit's](https://jovian-py.readthedocs.io/en/latest/jvn/commit.html) parameters to enable the user to commit with preferences like private notebook, new notebook project, to add outputs and files .....
 
-<img src="https://i.imgur.com/maHhYY2.png" class="screenshot" alt="jovian commit options jupyter extension" >
+Firstly, click the dropdown menu
 
-By default the parameters are derived from [jovian.commit](../jvn/commit), once the user changes a certain parameter they are retained.
+<img src="https://i.imgur.com/C4trCHn.png" class="screenshot" alt="jovian dropdown menu">
 
-<img src="https://i.imgur.com/2NKfNGB.png" class="screenshot" alt="jupyter extension commit options prompt" >
+Then, choose **commit with options**
+
+<img src="https://i.imgur.com/MAbD0aB.png" class="screenshot" alt="jovian commit options jupyter extension">
+
+By default the parameters are derived from [jovian.commit](https://jovian-py.readthedocs.io/en/latest/jvn/commit.html), once the user changes a certain parameter they are retained.
+
+<img src="https://i.imgur.com/3C7dokb.png" class="screenshot" alt="jupyter extension commit options prompt">
 
 Once the parameters are set to the need click `Commit` to the commit the notebook with these parameters set.
```

Comment:
Can you change this to `Click on `Commit` to commit the notebook with following options.`

Reply:
Sure